spec/runner.js

```diff
@@ -25,6 +26,32 @@ function decodeUTF8(decode) {
     return decode ? decode.toString('utf8') : null;
 }
 
+function cleanup() {
+    try {
+        run('rm -rf test*');
```

Comment:
Could you please use filesystem functions from fs? I think, someday we need to run this on Windows too ;)

Thanks :tada:

Reply:
@shalithasuranga Will do. Also, can you please check your discord dm quickly? It's a little urgent man.

Reply:
Will do.

Reply:
@shalithasuranga Will do. Also, can you please checkout your discord for once?